pandas/core/arrays/arrow/array.py

```diff
@@ -45,13 +46,22 @@
     from pandas.core.arrays.arrow._arrow_utils import fallback_performancewarning
     from pandas.core.arrays.arrow.dtype import ArrowDtype
 
+    ARROW_CMP_FUNCS = {
```

Comment:
isn't there a min version on some of these?

Reply:
Looks like they all have been around since pyarrow version 2 which I think is the version we require to use these functions

Reply:
Plus we catch the `except (pa.lib.ArrowNotImplementedError, pa.lib.ArrowInvalid)` when calling these with fallback behavior.